lib/js/components/Modal/Modal.vue

```diff
@@ -48,7 +48,7 @@
 
 	&__contentWrapper {
 		background-color: $color-default-background;
-		box-shadow: 0 4px 34px rgba(0, 0, 0, 0.85);
```

Comment:
is it ok, that DS component doesn't use shadow token?

Reply:
Fixed.